src/gpu/SDL_sysgpu.h

```diff
@@ -169,6 +169,8 @@ static inline Sint32 BytesPerImage(
 #define MAX_STORAGE_TEXTURES_PER_STAGE 8
 #define MAX_STORAGE_BUFFERS_PER_STAGE  8
 #define MAX_UNIFORM_BUFFERS_PER_STAGE  4
+#define MAX_COMPUTE_WRITE_TEXTURES     8
```

Comment:
Side note: we should probably prefix all these defines with `SDL_GPU_`

Reply:
True, will open a separate PR for that.